src/text.rs

```diff
@@ -355,13 +355,67 @@ pub fn draw_text_ex(text: &str, x: f32, y: f32, params: TextParams) {
                 dest_size: Some(vec2(dest.w, dest.h)),
                 source: Some(source),
                 rotation: angle_rad,
-                pivot: Option::Some(vec2(dest.x, dest.y)),
+                pivot: Some(vec2(dest.x, dest.y)),
                 ..Default::default()
             },
         );
     }
 }
 
+/// Draw multiline text with the given font_size, line_distance and color.
+/// If no line distance but a custom font is given, the fonts line gap will be used as line distance if it exists.
+pub fn draw_multiline_text(
+    text: &str,
+    x: f32,
+    y: f32,
+    font_size: f32,
+    line_distance: Option<f32>,
+    color: Color,
+) {
+    draw_multiline_text_ex(
+        text,
+        x,
+        y,
+        line_distance,
+        TextParams {
+            font_size: font_size as u16,
+            font_scale: 1.0,
+            color,
+            ..Default::default()
+        },
+    )
+}
+
+/// Draw multiline text with the given line distance and custom params such as font, font size and font scale.
+/// If no line distance but a custom font is given, the fonts line gap will be used as line distance if it exists.
+pub fn draw_multiline_text_ex(
+    text: &str,
+    x: f32,
+    mut y: f32,
+    line_distance: Option<f32>,
+    params: TextParams,
+) {
+    let font_line_distance = params
+        .font
+        .map(|font| {
```

Comment:
I feel like a few ifs might be a lot more appropriate here, but I would not focus on this until we will figure how to handle multiline gap correctly

Reply:
Changed it accordingly.

Reply:
Perfect, thanks for the change!